modules/core/shared/src/main/scala/lucuma/core/math/Angle.scala

```diff
@@ -12,125 +12,125 @@ import lucuma.core.optics._
 import monocle.{ Iso, Prism }
 
 /**
- * Exact angles represented as integral microarcseconds. These values form a commutative group over
- * addition, where the inverse is reflection around the 0-180° axis. The subgroup of angles where
- * integral microarcseconds correspond with clock microseconds (i.e., where they are evenly
- * divisible by 15 microarcseconds) is represented by the HourAgle subtype.
- *
+  * Exact angles represented as integral microarcseconds. These values form a commutative group over
+  * addition, where the inverse is reflection around the 0-180° axis. The subgroup of angles where
+  * integral microarcseconds correspond with clock microseconds (i.e., where they are evenly
+  * divisible by 15 microarcseconds) is represented by the HourAgle subtype.
+  *
  * Lawful conversion to and from other types/scales is provided by optics defined on the companion
- * object. Floating-point conversions are provided directly
- * @param toMicroarcseconds This angle in microarcseconds. Exact.
- */
+  * object. Floating-point conversions are provided directly
+  * @param toMicroarcseconds This angle in microarcseconds. Exact.
+  */
```

Comment:
Is this a scalafmt thing?

Reply:
Yes it is

Reply:
I think this should revert if you use the `scalafmt` we have commited in Explore and other projects.

Reply:
The conf we have in `lucuma-core` is identical to `explore`

Reply:
Ok. We don't have `scalafmt.conf` committed here in `lucuma-core` (it's actually in `.gitignore`) so each of us has their own version. So, just checking since yours might have been different.